docs/resources/appdata_dsource.md

```diff
@@ -0,0 +1,176 @@
+# Resource: <resource name> delphix_appdata_dsource
+
+In Delphix terminology, a dSource is a database that the Delphix Virtualization Engine uses to create and update virtual copies of your database. 
+A dSource is created and managed by the Delphix Continuous Data Engine.
+
+
+The dSource resource allows terraform to CREATE, READ and DELETE dSources. 
+Update operation is not supported. The supported parameters are listed below.
+
+## Example Usage
+dSource Linking can be done in 3 methods , the parametes for these methods wary based on choice.
+
+```hcl
+# Link dSource using external backup.
```

Comment:
FYI

We are organizing the names of our various Ingestion methods. External Backup, Delphix Ingestion, External Ingestion, etc Next time around, we will want to update these.